library/core/src/slice/mod.rs

```diff
@@ -1623,11 +1624,15 @@ impl<T> [T] {
     /// }
     /// ```
     #[unstable(feature = "slice_split_at_unchecked", reason = "new API", issue = "76014")]
+    #[rustc_const_unstable(feature = "const_slice_split_at_not_mut", issue = "none")]
```

Comment:
since the function itself is also unstable, point to the same issue and leave a comment on that issue so the information about the constness gets added to the main post

Reply:
Let me see if I get this right. For `split_at_unchecked` I'll change the feature to `"slice_split_at_unchecked"` and the issue to `"76014"`, and leave a comment on the issue.

But for the `split_at` function there still needs to be a new tracking issue `"const_slice_split_at_not_mut"`.

Reply:
Ah, good point, yea.